### DISCUSSION TITLE
Change Sample for addReproduciblePeakSet

### DISCUSSION THREAD

**Discussion by rhart604** [GreenleafLab/ArchR#1684] [Questions / Documentation] [answered]
I discovered that addReproduciblePeakSet is treating Sample as replicates. I tried to change Sample to reflect the actual grouping of my replicates but I see that I'm not allowed to change it. Is there a way to specify what metadata to treat as replicates for addReproduciblePeakSet? Or to change Sample?

**Comment by rcorces** [comment, marked as answer]
If I'm understanding your question correctly, as of `release_1.0.2`, this is already possible. However, it happens during the `addGroupCoverages()` step. see the `sampleLabels` param:

https://github.com/GreenleafLab/ArchR/blob/f6c0388bd37023400794c9ae8562ad69e3ba9fd7/R/GroupCoverages.R#L9-L12
